Packs/GoogleCloudLogging/README.md

```diff
@@ -1,6 +1,22 @@
-Google Cloud Logging is a service provided by Google Cloud Platform (GCP) that allows you to store, search, analyze, monitor, and alert on log data and events from your applications and services running on GCP.
-The Google Cloud Logging Cortex XSOAR pack helps users to centralize all their logs in a single location, making it easier to troubleshoot issues and gain insights from their data.
 ## What does this pack do
-This integration enables you to:
 
-- Retrieve log entries that originated from a project/folder/organization/billing account.
+The Google Cloud Logging Cortex <~XSOAR>XSOAR</~XSOAR><~XSIAM> <~XSIAM>XSIAM</~XSIAM> pack helps users to centralize all their GCP logs in a single location, making it easier to troubleshoot issues and gain insights from their data: 
```

Comment:
```suggestion
The Google Cloud Logging Cortex <~XSOAR>XSOAR</~XSOAR><~XSIAM>XSIAM</~XSIAM> pack helps users to centralize all their GCP logs in a single location, making it easier to troubleshoot issues and gain insights from their data: 
```

Reply:
@ShirleyDenkberg Thanks for the quick review. 
I inserted some reorganization and reformatting of the sections to better emphasize the separation between the Integration to the Modeling Rule & Parsing Rule which are independent of the integration. If you could re-review the updated version as well.

Reply:
@ShirleyDenkberg Thanks for the quick review. 
I inserted some reorganization and reformatting of the sections on the README and release notes to better emphasize the separation between the Integration to the Modeling Rule & Parsing Rule which are independent of the integration. If you could re-review the updated version as well.